install/playbooks/roles/access-check/templates/postmaster-ip-whitelist.cf

```diff
@@ -0,0 +1,9 @@
+# Postmaster specific whitelist file.
+# Lines starting with '#' are ignored
+# You can use ipcalc or whois tools to create CIDR notations
+
+# Whitelist LAN and local IP addresses
+192.168.0.0/16
+172..16.0.0/12
+10.10.0.0/8
```

Comment:
Typos: `172.16.0.0/12` and `10.0.0.0/8`.

Reply:
Thanks